### PR TITLE
Add demo and element tags for docs

### PR DESCRIPTION
**Pull request by e111077**
Missing the little demo button thing in [the docs](https://elements.polymer-project.org/elements/prism-element). :(

### CODE REVIEW

**Comment by notwaldorf** [comment]
:shipit: but you have a typo

File: prism-highlighter.html [hunk 1]
```diff
@@ -24,6 +24,9 @@
 `lang` property, containing a string like `"html"`, `"js"`, etc.
 
 This flow is supported by [`<marked-element>`](https://github.com/PolymerElements/marked-element).
+
+@element prisim-highlighter
```
Comment (notwaldorf) [inline prism-highlighter.html]: typo! prism!

**Comment by FredKSchott** [comment]
@e111077 2 lines, 3 commits. The math checks out :)

**Comment by e111077** [comment]
@FredKSchott I'm edumacated!

**Pull request merged**